cce/clitse/cli.go

```diff
@@ -13,23 +14,49 @@ import (
 	"path"
 	"path/filepath"
 	"strings"
+	"time"
 
 	"github.com/labstack/echo"
 )
 
+const (
+	port      = 9999
+	timeLimit = time.Second * 15
+)
+
+var (
+	client = &http.Client{
+		Timeout: time.Second * 40,
```

Comment:
Tem mais coisa a olhar que isso: https://blog.cloudflare.com/the-complete-guide-to-golang-net-http-timeouts/